index.html

```diff
@@ -18,23 +18,23 @@
   <title>My Portfolio</title>
 </head>
 <body>
-  <header>
-    <a href="#" class="logo">John Doe</a>
+  <header class="d-flex font-inter">
+    <a href="#" class="text-white">John Doe</a>
     <nav>
       <span id="bars">
         <i class="fas fa-bars"></i>
       </span>
     </nav>
   </header>
-  <section class="headline-section">
+  <section class="headline-section d-flex secondary-color">
     <div class="headline-text-contents">
-      <h1>Hey There. <br>I&apos;m Alex</h1>
+      <h1 class="font-crete-round text-xl primary-text">Hey There. <br>I&apos;m Alex</h1>
       <h5>I am a Software Developer</h5>
-      <p>
+      <p class="text-lg">
         I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hestiate to contact me.
       </p>
-      <div class="sider-icons-menu">
+      <div class="sider-icons-menu text-white font-size-medium">
         <ul>
           <li><i class="fab fa-github icons"></i></li>
           <li><i class="fab fa-linkedin-in"></i></li>
```

Comment:
- The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).

Reply:
Well solved